index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="main-nav">
+            <nav>
+                <div class="left-bar">
+                    <p class="time">9:41</p>
+                    <p class="name">Oscar Byiringiro</p>
+                </div>
+                <div class="right-bar">
+                    <div class="top-icons">
+                        <img src="./images/Cellular Connection.png">
+                        <img src="./images/Wifi.png">
+                        <img src="./images/Battery.png">
+                                    
+                                
+                    </div>
+                    <div class="bottom-icons">
+                        <img src="./images/Union.png">
+                            
+                    </div>
+                    
+                        
+                </div>
+            </nav>
+        </header>
+        <section class="main-body">
+            <div class="container">
+                <div class="intro">
+                    <h1>Hey There</h1>
+                    <h1>I'm Oscar</h1>
+                </div>
+                <div class="title">
+                    <p>I'am a software developer</p>
```

Comment:
- [ ] Kindly use the h2 or h3... here because it will increase your website's semantics, and the screen readers will thank you 😄